tools/data/jhmdb/preparing_jhmdb.md

```diff
@@ -0,0 +1,82 @@
+# Preparing JHMDB
+
+For basic dataset information, you can refer to the dataset [website](http://jhmdb.is.tue.mpg.de/).
+Before we start, please make sure that the directory is located at `$MMACTION2/tools/data/jhmdb/`.
+
+## Download and Extract
+
+You can download the RGB frames, optical flow and ground truth annotations from [google drive](https://drive.google.com/drive/folders/1BvGywlAGrACEqRyfYbz3wzlVV3cDFkct).
+The data are provided from [MOC](https://github.com/MCG-NJU/MOC-Detector/blob/master/readme/Dataset.md), which is adapted from [act-detector](https://github.com/vkalogeiton/caffe/tree/act-detector).
+
+After downloading the `JHMDB.tar.gz` file and put it in `$MMACTION2/tools/data/jhmdb/`, you can run the following command to extract.
+
+```shell script
+tar -zxvf JHMDB.tar.gz
+```
+
+If you have plenty of SSD space, then we recommend extracting frames there for better I/O performance.
+
+You can run the following script to soft link SSD.
+
+```shell script
+# execute these two line (Assume the SSD is mounted at "/mnt/SSD/")
+mkdir /mnt/SSD/JHMDB/
+ln -s /mnt/SSD/JHMDB/ ../../../data/jhmdb
```

Comment:
where to run this line?

Reply:
As mentioned above, the directory is located at `$MMACTION2/tools/data/jhmdb/`.